src/coreclr/jit/lowerxarch.cpp

```diff
@@ -3275,6 +3275,30 @@ void Lowering::LowerHWIntrinsicGetElement(GenTreeHWIntrinsic* node)
     GenTree* op1 = node->Op(1);
     GenTree* op2 = node->Op(2);
 
+    if (op2->IsIntegralConst(0))
```

Comment:
This method should have an `assert(intrinsicID = GetElement)`

Reply:
Fixed.